examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,167 @@
+#The software contained in this file is free and unencumbered software released into the public domain.
+#Anyone is free to use the software contained in this file as they choose, including incorporating
+#it into proprietary software.
+import time
+import sys
+import threading
+from pygstc.gstc import GstcError, GstdError, GstdClient
+from pygstc.logger import CustomLogger
+
+class GstcPlayer:
+
+  def __init__(self):
+    #Create a custom logger with loglevel=DEBUG
+    self.gstd_logger = CustomLogger('simple_playback', loglevel='DEBUG')
+    #Create the client with the logger
+    self.gstc = GstdClient(logger=self.gstd_logger)
+    self.pipeline = "fakesrc ! fakesink"
+
+    #Check if pipe is already created
+    if ( self.pipe_exists("p0")):
+      self.gstc.pipeline_delete("p0")
+    self.pipe_created = False
+
+    #Error handler thread
+    self.running = True
```

Comment:
I think you can factor the code in a way that this variable is not needed. You may exit the loop when the bus read fails because the pipeline no longer exists.